hadoop-common-project/hadoop-common/pom.xml

```diff
@@ -1093,8 +1093,9 @@
                         <goal>exec</goal>
                     </goals>
                     <configuration>
-                        <executable>${basedir}/../../dev-support/bin/releasedocmaker</executable>
+                        <executable>${shell-executable}</executable>
```

Comment:
where does this get set?

Reply:
This needs to be supplied as an mvn arg. Please refer to my changes in `BUILDING.txt` in this PR for the sample invocation.

Reply:
the default value is there in the hadoop-main pom.xml
https://github.com/apache/hadoop/blob/trunk/pom.xml#L123

the value is bash by default, for windows it needs to be overridden by some diff value as per Building.txt